dump/create_pel.hpp

```diff
@@ -35,6 +35,17 @@ uint32_t createSbeErrorPEL(const std::string& event, const sbeError_t& sbeError,
                            const FFDCData& ffdcData,
                            const Severity& severity = Severity::Error);
 
+/**
```

Comment:
This has exactly the same definition as the existing function, so can't we make it a common function?

Reply:
For createSbeErrorPEL severity is set by the caller and for createPOZSbeErrorPEL severity is set by the ffdc data packet data